### PR TITLE
Comment Class Improvements

### PR DESCRIPTION
**Pull request by dshanske**
* Removes h-as properties
* Addresses fact that list_linkbacks was not using comment_class filter
* Introduces type argument in list_linkbacks that allows a list and single element class to be generated by default.
* Remove unused properties

### CODE REVIEW

**Comment by dshanske** [comment]
Just realized I didn't change the unit tests

File: templates/linkbacks.php [hunk 1]
```diff
@@ -91,10 +91,7 @@
 	<?php
 	list_linkbacks(
 		array(
-			'li-class' => array(
-				'single-mention',
-				'p-rsvp',
-			)
+			'tag',
```
Comment (pfefferle) [inline templates/linkbacks.php]: Shouldn‘t this be `type = rsvp-yes`?

Comment (dshanske) [reply templates/linkbacks.php]: Correct - fixed

File: templates/linkbacks.php [hunk 2]
```diff
@@ -91,10 +91,7 @@
 	<?php
 	list_linkbacks(
 		array(
-			'li-class' => array(
-				'single-mention',
-				'p-rsvp',
-			)
+			'rsvp-yes',
```
Comment (pfefferle) [inline templates/linkbacks.php]: You still forgot `type =>`!

**Comment by dshanske** [comment]
Sorry about that. Fixed now

**Comment by pfefferle** [comment]
Thanks!

**Pull request merged**